### PR TITLE
Update quick-start.md

### PR DESCRIPTION
**Pull request by HendrikRunte**
Fixing minor typos

## Changes

-- Only typos fixed

## Testing

typo fixes only

## Docs

bug fix only

### CODE REVIEW

**Comment by borisv** [comment]
You are using an outdated version, the current version is PR 1830. Most of your changes have already been discussed there. Please review those versions. There's actually the complete first two sections of the docs to review. You'll find the explanation for the use of lowercase lettering for 'du' there as well.

https://github.com/withastro/astro/pull/1830

File: docs/src/pages/de/quick-start.md [hunk 1]
```diff
@@ -30,46 +30,46 @@ npm run build
 
 Um mehr über den Installationsprozess und den ersten Gebrauch von Astro zu lernen, [lies unseren Installations-Leitfaden.](de/installation)
 
-Wenn du es bevorzugst anhand eines Beispiels zu lernen, schau dir die [komplette Liste an Beispielen](https://github.com/withastro/astro/tree/main/examples) auf GitHub an.
-Du kannst jedes Beispiel ausprobieren, indem du `npm init astro -- --template "EXAMPLE_NAME"` ausführst.
+Wenn du es bevorzugst, anhand eines Beispiels zu lernen, schau Dir die [komplette Liste an Beispielen](https://github.com/withastro/astro/tree/main/examples) auf GitHub an.
```
Comment (drwpow) [inline docs/src/pages/de/quick-start.md:33]: I believe only `Sie` gets capitalized; `dir`, `deine`, etc. do not. I don’t think most of these are typos.

My German is too bad to verify the other word changes, though—they may still be valid. Would love a review from someone fluent.

Comment (drwpow) [reply docs/src/pages/de/quick-start.md:33]: Oops! Missed @borisv’s comment above.

I’m going to close this PR in favor of #1830 to cut down on possible confusion (please move any suggestions for change to that PR!)

**Comment by drwpow** [comment]
Closing in favor of #1830. Please request changes there.

**Pull request closed without merging**